apps/omg_watcher/test/fixtures.exs

```diff
@@ -227,6 +235,11 @@ defmodule OMG.Watcher.Fixtures do
         eth_height: 1
       })
 
+    {:ok, {_, _, db_updates}, _} =
+      OMG.State.Core.form_block(0, %OMG.State.Core{pending_txs: Enum.reverse(recovered_txs), height: blknum})
```

Comment:
don't use such deep internals in testing. What if one decides to keep `pending_txs` already reversed?!

If I was forced to have those `db_updates` I'd go through a couple of **public** calls from `OMG.State.Core`, or even just write them out manually.

But see my other comment - I honestly think this is not necessary here.